### ISSUE TITLE
Train the network

### ISSUE THREAD

**Issue by Zermy** [sagr4019/ResearchProject#166]
- Train the network and show the results.
- Compare results with other team members
- Vary the training data

**Comment by Zermy** [comment]
First results with 5000 data (invalid/valid) and 10 epochs:
`
Computing training accuracy...
Accuracy: 0.5419847328244275
              precision    recall  f1-score   support

     invalid       0.59      0.17      0.27      1267
       valid       0.53      0.89      0.67      1353

   micro avg       0.54      0.54      0.54      2620
   macro avg       0.56      0.53      0.47      2620
weighted avg       0.56      0.54      0.47      2620
`

**Comment by Zermy** [comment]
100 Epochs 10000 Data

```
Computing training accuracy...
Accuracy: 0.5312143129044538
              precision    recall  f1-score   support

       valid       0.53      0.57      0.55      2624
     invalid       0.53      0.50      0.51      2630

   micro avg       0.53      0.53      0.53      5254
   macro avg       0.53      0.53      0.53      5254
weighted avg       0.53      0.53      0.53      5254

```

**Comment by Zermy** [comment]
/home/timm/.local/lib/python3.5/site-packages/sklearn/metrics/classification.py:1143: UndefinedMetricWarning: Precision and F-score are ill-defined and being set to 0.0 in labels with no predicted samples.

**Comment by Zermy** [comment]
The bad results may be because we are training the net with too little data. Unfortunately sagr4019 and I are not able to use more than 30000 data, because the RAM is not enough for a larger amount of data. We currently have 16 GB Ram in use and no access to a more powerful machine.

**Comment by Zermy** [comment]
Accuracy: 0.5999065202150035
              precision    recall  f1-score   support

       valid       0.61      0.91      0.73      7725
     invalid       0.49      0.13      0.21      5112

   micro avg       0.60      0.60      0.60     12837
   macro avg       0.55      0.52      0.47     12837
weighted avg       0.56      0.60      0.52     12837

I get 0.59 accuracy with 50 epochs, 30k valid and 20 invalid data and a learn-rate of 0,001.
